tests/test_plugin.py

```diff
@@ -56,15 +56,19 @@ def expected_log_line(track, challenge):
         actual = [(r.name, r.levelname, r.message) for r in caplog.records if "esrally race" in r.message]
         assert actual == expected
 
-    def test_runs_correct_install_command(self, caplog, temp_repo, distribution_version, revision, run, example):
+    def test_runs_correct_install_command(self, caplog, temp_repo, distribution_version, revision, source_build_release, run, example):
         install_option = f'--distribution-version={distribution_version}' if distribution_version else f'--revision={revision}'
+        if '--revision' in install_option and source_build_release:
+            install_option += ' --source-build-release'
```

Comment:
This does not cycle through different combination of options and relies on a user to provide them when running `pytest`. Please use [parametrize](https://docs.pytest.org/en/stable/how-to/parametrize.html#parametrizemark) instead.